leveler/commands/lvladmin/settings.py

```diff
@@ -25,6 +25,7 @@ async def overview(self, ctx):
         em = discord.Embed(colour=await ctx.embed_color())
         settings = {
             "Enabled": self.bool_emojify(not await self.config.guild(ctx.guild).disabled()),
+            "Anti-Rep Farm": self.bool_emojify(await self.config.guild(ctx.guild).antirepfarm()),
```

Comment:
Again, not really selfdescriptive
```suggestion
            "Rep users rotation": self.bool_emojify(await self.config.guild(ctx.guild).rep_rotation()),
```